palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/service/UserService.java

```diff
@@ -24,35 +23,30 @@
 import uk.gov.gchq.palisade.service.palisade.request.GetUserRequest;
 import uk.gov.gchq.palisade.service.palisade.web.UserClient;
 
-import java.net.URI;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 import java.util.concurrent.Executor;
-import java.util.function.Supplier;
 
 public class UserService implements Service {
-
     private static final Logger LOGGER = LoggerFactory.getLogger(UserService.class);
-    private final UserClient userClient;
-    private final Supplier<URI> uriSupplier;
+
+    private UserClient client;
+
     private final Executor executor;
 
-    public UserService(final UserClient userClient, final Supplier<URI> uriSupplier, final Executor executor) {
-        this.userClient = userClient;
-        this.uriSupplier = uriSupplier;
+    public UserService(final UserClient userClient, final Executor executor) {
+        this.client = userClient;
         this.executor = executor;
     }
 
     public CompletableFuture<User> getUser(final GetUserRequest request) {
         LOGGER.debug("Getting user from user service: {}", request);
-
+        LOGGER.info("UserClient now is: {}", client);
```

Comment:
Does this ginve any meaningful info? Probably better as debug

Reply:
removed

Reply:
can you double check, I cant see the change in the files changed